nextcord/message.py

```diff
@@ -781,6 +785,12 @@ def __init__(
             except KeyError:
                 continue
 
+        self.interaction: Optional[MessageInteraction] = (
+            MessageInteraction(data.get("interaction"))
```

Comment:
You made data a keyword argument and state also need state to call the constructor. Possibly something like this?
```suggestion
            MessageInteraction(data=data.get("interaction"), state=self._state)
```